.github/workflows/ci-pipline.yml

```diff
@@ -32,8 +32,9 @@ jobs:
       with:
         username: 'aliwen'
         password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
-    - run: docker build --target production --tag aliwen/todo_app:$GITHUB_SHA .
-    - run: docker push aliwen/todo_app:$GITHUB_SHA
+    - run: docker build --target production --tag aliwen/todo_app:latest .
+    - run: docker push aliwen/todo_app:latest
+    - run: curl -dH -X POST ${{secrets.WEB_HOOK}}
```

Comment:
One minor suggestion, I'd recommend adding the `--fail` flag here as otherwise curl will report success if it gets a response _even if the response itself is a failure like a 404/500_